services/director-v2/tests/unit/conftest.py

```diff
@@ -51,8 +51,11 @@ def simcore_services_network_name() -> str:
 
 @pytest.fixture
 def simcore_service_labels() -> SimcoreServiceLabels:
+    json_schema_extra = {}
+    SimcoreServiceLabels.model_json_schema()(json_schema_extra)
```

Comment:
this is rightfully making your tests fail. Please see director-v2 unit test. Dict is not callable.

Reply:
indeed.  I was very sleepy :-)
NOTE that I do not call anymore `model_config` to get the example anymore but instead ask the class to build the json-schema for me via model_json_schema. This comes with some advantages and avoids the battery of asserts you did not like :-)